src/routes/sports.ts

```diff
@@ -16,23 +16,19 @@ import {
 
 const router = express.Router();
 
-// GET: Recibe todo lo que contiene el Sport
 router.get('/sports', getSports);
 
-// TODO: endpoint to upload image to GCP before createPlan on POST METHOD
```

Comment:
As you can see here, we are on sports route but you are talking on this comment about plan, this is because you are just copy and paste the comments, if you feel the comment is repetitive, so don't do it
by the way I'm not against copy and paste but read at least what you are copy and paste